charter/charter-detecting-unwanted-location-trackers.md

```diff
@@ -47,7 +47,11 @@ The WG is expected to:
 5. Design mechanisms to ensure that devices that do not correctly implement or adhere to the DULT protocol can be detected and excluded from being trackable via crowdsourced location networks.
  * This includes considerations for addressing legacy trackers that cannot update to the DULT protocol.
 
-The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
+The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on small and not easily-discoverable accessories. Accessories are considered easily discoverable if they meet one of the following criteria:  
```

Comment:
I think this "end-to-end" language is a bit confusing. Let me see if I can flesh out the intention here. 

- Standardize the protocols used by (1) the tag to communicate with finding devices (2) the tag to communicate with devices when apparent unwanted tracking is detected (e.g., to cause it to beep).
- Not standardize the protocols use by the finding device to report to the tracking network (e.g., to upload to iCloud), the protocols used to initialize the tag, or the protocols used by the tag owner to query the network for location.

One way to phrase this is that it ought to be enough to allow you to build your own tag that would work with the network, assuming it had the right keying material. 

Does that sound right?